.github/pull_request_template.md

```diff
@@ -1,38 +1,33 @@
-Description
------------
+<!--
+⚠️ Your pull request title should be short, detailed, and understandable for all.
+⚠️ If your pull request fixes an open issue, please link to the issue.
 
-Please explain the changes you made here.
+Ensure you have completed the following before continuing.
+✅ I have read the CONTRIBUTING document.
+✅ I have added tests to cover my changes.
+✅ I updated the documentation, where relevant.
+✅ I used type hints (https://www.python.org/dev/peps/pep-0484/) in function signatures.
+✅ I used Google-style (https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) docstrings for functions.
+✅ I added myself / the copyright holder to the AUTHORS file.
+-->
 
-Checklist
----------
+<!--
+If the validation checks fail
+  1. Run `make check-types` (from the root directory of the repository) and fix any mypy (https://mypy.readthedocs.io/en/stable/) errors.
 
-Check off the following once complete (or if not applicable) after opening the PR. The PR will be reviewed once this checklist is complete and all tests are passing.
+  2. Run `make check-style` and fix any flake8 (http://flake8.pycqa.org) errors.
```

Comment:
It might be good to add a comment about ignoring any errors in `docs`. I had an issue where I ran it locally and ended with a long list of errors in folders where I did not make any changes. 

Seems Github workflows ignore this check for `docs`.

Reply:
Hi Purva, thanks for the comment. I'm not sure I'm able to reproduce. Running `flake8`/`make check-style` seem to ignore the `docs` directory for me. Can you confirm that when you run `flake8` you get errors from the `docs` directory?

Reply:
Per later comment on Discord seems the workflow checks style in mitiq/mitiq, so sounds like we don't need to mention ignoring local errors after all.

Reply:
Yes, the Github workflow does ignore docs folder but if I follow[ the style guidelines instructions in documentation](https://mitiq.readthedocs.io/en/stable/contributing.html#style-guidelines) and check them on my laptop's virtual environment, I end up with a long list of errors. 

I am not too sure about the difference between `make check-style` and `flake8/make check-style` but the latter does not work on my laptop. I get `bash: cd: flake8: No such file or directory`. 

[docs_style_errors.txt](https://github.com/unitaryfund/mitiq/files/9006485/docs_style_errors.txt)

Reply:
That's strange. What OS are you on? Are you running `make check-style` on your computer, or in a container/VM?

The [definition](https://github.com/unitaryfund/mitiq/blob/0af906327282fa2acd09a82fedffbc080b4d19bb/Makefile#L15-L16) of `make check-style` simple runs the `flake8` command. If you can't run `flake8` directly, you shouldn't be able to run `make check-style` either (or at least I don't understand how that would work).

Reply:
I am on linux-mint. I have a mitiq virtual environment that I use to run tests etc. The virtual environment was installed via `pip install virtualenv`. 

I do have `flake8` installed in my environment and I can run `make check-style`. I think I got confused when I saw `flake8/make check-style` because `/` makes it look like a directory path. You were saying use either `flake8` or `make check-style`, right ? Both lead to errors in `docs`. My workflow is below :

```python
source mitiq_venv/bin/activate
cd mitiq
flake8 # can run and won't skip docs folder
make check-style # same as above
```